packages/jaeger-ui/src/utils/config/get-config.test.js

```diff
@@ -22,19 +22,30 @@ import defaultConfig, { deprecations } from '../../constants/default-config';
 
 describe('getConfig()', () => {
   let oldWarn;
+  let oldWindowGetJaegerUiConfig;
   let warnFn;
 
-  beforeEach(() => {
+  beforeAll(() => {
     oldWarn = console.warn;
+    oldWindowGetJaegerUiConfig = window.getJaegerUiConfig;
```

Comment:
Seems like the change in `setupTests` will cause the old function to be the no op?

Reply:
yeah it does, but `get-config.test.js` doesn't need to worry about the global defined in `setupTests`. Should that change for whatever reason, we wouldn't want to have to change it here too.

Reply:
Seems as those this store-n-restore is wholly unnecessary as `setupTests.js` is run before each file!
Just learned that from trial and error and [here](https://jestjs.io/docs/en/configuration#setupfilesafterenv-array)!